trailblazer/constants.py

```diff
@@ -3,9 +3,12 @@
 
 COMPLETED_STATUS = "completed"
 FAILED_STATUS = "failed"
+HOURS_IN_DAY: int = 24
+MINUTES_PER_HOUR: int = 60
 ONGOING_STATUSES = ("pending", "running", "error", "completing")
 ONE_MONTH_IN_DAYS: int = 31
 PRIORITY_OPTIONS = ("low", "normal", "high", "express", "maintenance")
+SCALE_SECONDS_TO_MINUTES: int = 60
```

Comment:
```suggestion
HOURS_IN_DAY: int = 24
MINUTES_PER_HOUR: int = 60
ONGOING_STATUSES = ("pending", "running", "error", "completing")
ONE_MONTH_IN_DAYS: int = 31
PRIORITY_OPTIONS = ("low", "normal", "high", "express", "maintenance")
SECONDS_PER_MINUTE: int = 60
```
why the different naming styles?

Reply:
Fixed